apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -266,7 +305,28 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
         return null;
     }
 
-    private addContentlet($event: any): void {
+    private addContentType($event: any): void {
+        const container: DotPageContainer = {
+            identifier: $event.data.container.dotIdentifier,
+            uuid: $event.data.container.dotUuid
+        };
+        this.dotEditContentHtmlService.setContainterToAppendContentlet(container);
+        this.dotContentletEditorService
+            .getActionUrl($event.data.contentType.variable)
+            .pipe(take(1))
+            .subscribe((url) => {
+                this.dotContentletEditorService.create({
+                    data: { url },
+                    events: {
+                        load: (event) => {
+                            event.target.contentWindow.ngEditContentletEvents = this.dotEditContentHtmlService.contentletEvents$;
```

Comment:
Isn't this injected already?

Reply:
no, is needed every time the create content dialog is opened, to inject that to the iframe.